server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const server = express();
+const projectsRoute = require('./projects/projects-router.js')
+const actionsRoute = require('./actions/actions-router.js')
+
+server.use(express.json());
+server.use('/api/projects', projectsRoute);
+server.use('/api/actions', actionsRoute)
+
+
```

Comment:
Sanity check here would be nice. Otherwise, this server.js file is fine :heavy_check_mark: